logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/MLSConversationRepository.kt

```diff
@@ -423,6 +424,14 @@ class MLSConversationDataSource(
             }
         }
 
+    override suspend fun getConversationVerificationStatus(groupID: GroupID): Either<CoreFailure, MLSVerificationStatus> =
+        mlsClientProvider.getMLSClient().flatMap { mlsClient ->
+            wrapMLSRequest { mlsClient.getGroupVerify(idMapper.toCryptoModel(groupID)) }
+        }.map {
+            if (it) MLSVerificationStatus.VERIFIED
```

Comment:
```suggestion
            if (it) ConversationVerificationStatus.VERIFIED
```
Since we need to use this object for Proteus as well. let's have it not only for MLS/

Reply:
A `ConversationVerificationStatus`, which can be the same for MLS and for Proteus, either `VERIFIED`, `DEGRADED`, and `NOT_VERIFIED`.


Will there be a `MLSVerificationDetails`, which may have MLS-specific stuff? I guess Proteus won't have it.

Reply:
I thought we'll get a separate object for Proteus verification 🤔 as we need to distinguish them

Reply:
If this is just the high-level status, it'll be the same between Proteus and MLS, we could rename it as Mojtaba said.

If this will contain MLS-specific stuff about E2EI, then I'd suggest to keep this way. But considering it's just an `enum`, I think it will be the same as Proteus' 🤔